src/examples/libvmem/libart/art.c

```diff
@@ -207,9 +207,8 @@ find_child(art_node *n, unsigned char c)
 		}
 		break;
 
-	{
-	__m128i cmp;
-	case NODE16:
+	case NODE16: {
+		__m128i cmp;
```

Comment:
For consistency sake you should either declare this variable before the switch or add brackets to every case.